source/agora/common/Config.d

```diff
@@ -488,7 +485,7 @@ unittest
                  PublicKey.fromString("GBYK4I37MZKLL4A2QS7VJCTDIIJK7UXWQWKXKTQ5WZGT2FPCGIVIQCY5"),
                  PublicKey.fromString("GBYK4I37MZKLL4A2QS7VJCTDIIJK7UXWQWKXKTQ5WZGT2FPCGIVIQCY5")])])]);
 
-    assert(quorum == expected);
+    assert(cast()quorum == expected);
```

Comment:
I guess you can't make `expected` immutable ?

Reply:
Doh, of course.